controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/StreamRequestHandler.java

```diff
@@ -65,31 +65,51 @@ public boolean toPostpone(ControllerEvent event, long pickupTime, Throwable exce
 
     @Override
     public CompletableFuture<Void> processScaleOpRequest(ScaleOpEvent scaleOpEvent) {
+        log.info("processing scale request {} for stream {}/{}", scaleOpEvent.getRequestId(), scaleOpEvent.getScope(), scaleOpEvent.getStream());
         return withCompletion(scaleOperationTask, scaleOpEvent, scaleOpEvent.getScope(), scaleOpEvent.getStream(),
-                OPERATION_NOT_ALLOWED_PREDICATE.or(e -> e instanceof EpochTransitionOperationExceptions.ConflictException));
+                OPERATION_NOT_ALLOWED_PREDICATE.or(e -> e instanceof EpochTransitionOperationExceptions.ConflictException))
+                .thenAccept(v -> {
+                    log.info("processing scale request {} for stream {}/{} complete", scaleOpEvent.getRequestId(), scaleOpEvent.getScope(), scaleOpEvent.getStream());
+                });
     }
 
     @Override
     public CompletableFuture<Void> processUpdateStream(UpdateStreamEvent updateStreamEvent) {
+        log.info("Processing update request {} for stream {}/{}", updateStreamEvent.getRequestId(), updateStreamEvent.getScope(), updateStreamEvent.getStream());
         return withCompletion(updateStreamTask, updateStreamEvent, updateStreamEvent.getScope(), updateStreamEvent.getStream(),
-                OPERATION_NOT_ALLOWED_PREDICATE);
+                OPERATION_NOT_ALLOWED_PREDICATE)
+                .thenAccept(v -> {
+                    log.info("Processing update request {} for stream {}/{}", updateStreamEvent.getRequestId(), updateStreamEvent.getScope(), updateStreamEvent.getStream());
```

Comment:
We could update it to "Processing update request {} for stream {}/{} complete."
The same comment applies to the below methods.